Assets/MixedRealityToolkit.Tests/PlayModeTests/VisualThemeTests.cs

```diff
@@ -28,6 +28,8 @@ namespace Microsoft.MixedReality.Toolkit.Tests
 {
     class VisualThemeTests
     {
+        private const string DefaultColorProperty = "_Color";
```

Comment:
We're going through this with pointers now, we will probably add a Pointers folder under tests (getting lots of tests related to pointers). Probably makes sense to add a folder for Interactable. Not required, just a suggestion.

Reply:
I'd actually say this shouldn't be under the interactable folder. The idea is to decouple visual themes from that feature. But good point on tests in MRTK overall could be better organized